src_bonus/main.cpp

```diff
@@ -0,0 +1,41 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   main.cpp                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2024/03/13 13:13:11 by mcourtoi          #+#    #+#             */
+/*   Updated: 2024/03/14 18:56:14 by mcourtoi         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "class/Bot.hpp"
+#include <iostream>
+#include <cstdlib>
+
+int main(int const ac, char const *const *const av)
+{
+	if (ac < 3)
```

Comment:
You'd better check that `ac == 3` instead of checking that `ac >= 3`, don't you think?